features/steps/v6uatsteps.rb

```diff
@@ -0,0 +1,30 @@
+require 'date'
+
+# Number
+
+Then(/^the payload field "(.+)" is a number$/) do |field_path|
+    value = read_key_path(Server.current_request[:body], field_path)
+    assert_kind_of Numeric, value
```

Comment:
The change to `assert_kind_of` should have been applied to the other blocks as well, but I can take care of that when I move it into MR.

Reply:
NP, I've made the change.